src/components/mdx-img/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactElement } from 'react'
+import { MdxImgProps } from './types'
+import classNames from 'classnames'
+import s from './mdx-img.module.css'
+
+function MdxImg({
+  src,
+  alt,
+  title,
+  noMargin,
+  noBorder,
```

Comment:
What are some of the use cases for these two props?

Reply:
Didn't see the comment explaining these elsewhere because the `Files` view changed as I was reviewing, but resolving because I've since found my answer in another comment. 😊